SS_benchfore.tpl

```diff
@@ -1664,14 +1668,14 @@ FUNCTION void Get_Benchmarks(const int show_MSY)
 
     Btgt_Fmult2 = Fmult;
     if (rundetail > 0 && mceval_counter == 0 && show_MSY == 1)
-      cout << " got_F_Blimit " << Btgt_Fmult2 << " " << Btgt2 / Blim_report << endl;
+      echoinput << "Calculated F_Blimit " << Btgt_Fmult2 << " " << Btgt2 / Blim_report << endl;
     Mgmt_quant(18) = Btgt2;
     Mgmt_quant(19) = equ_F_std;
     Mgmt_quant(20) = sum(equ_catch_fleet(2)) * Equ_SpawnRecr_Result(2);
   } //  end finding F for Blimit
 
   if (rundetail > 0 && mceval_counter == 0 && show_MSY == 1)
-    cout << " got Fmsy " << MSY_Fmult << " " << MSY << endl;
+    echoinput << "Calculated Fmsy " << MSY_Fmult << " " << MSY << endl;
```

Comment:
Needs curly braces.

Reply:
Kelli,
The style guide we are using does not require {} for single statement conditionals

Reply:
Okay, thanks for the clarification Rick, I will remove my comments below and maybe add some language to the document Neal started because right now it says  "Curly braces should always be used."